### PR TITLE
Remove reliance on createdbyuserid

### PR DESCRIPTION
**Pull request by sbinlondon**
* Removes `createdByUser` from GraphQL queries
* Removes `createdByUser.id` from any `LoggedInUser` functions
* Updates any tests that were broken

### CODE REVIEW

**Comment by sbinlondon** [comment]
Hm looks like some tests are failing again. When I ran them locally, the signin tests were also failing for some reason.

I'll look into them but this might run over into next week.

**Review by znarf — APPROVED** [review]
Looks good to me, would be great to have a second review from @Betree before merging.

File: lib/__tests__/LoggedInUser.test.js
Comment (Betree) [inline lib/__tests__/LoggedInUser.test.js:133]: We can remove this one, it's now the same than the line before 

```suggestion
```

File: lib/__tests__/LoggedInUser.test.js
Comment (Betree) [inline lib/__tests__/LoggedInUser.test.js:148]: Same here

```suggestion
```

File: test/cypress/integration/30-archiveCollective.test.js
Comment (Betree) [inline test/cypress/integration/30-archiveCollective.test.js:15]: :ice_cream: maybe simpler to put everything in a single `before`?

```suggestion
  before(() => {
    cy.createCollective({ type: 'ORGANIZATION' }).then(({ slug }) => {
      orgSlug = slug;
    });
    
    cy.createCollective({ type: 'COLLECTIVE' }).then(({ slug }) => {
      collectiveSlug = slug;
    });
  });
```

**Review by Betree — COMMENTED** [review]
I don't really understand the reason for the e2e test changes, but the code seems to do the job :+1:

**Comment by sbinlondon** [comment]
> I don't really understand the reason for the e2e test changes, but the code seems to do the job 👍

They were failing. I notice lately the tests that are structured like:

```
cy.login();
cy.createHostedCollective().then(({ slug }) => {
    // do all the stuff
});
```

vs

```
let orgSlug

 before(() => {
    cy.createCollective({ type: 'ORGANIZATION' }).then(({ slug }) => {
      orgSlug = slug;
    });
cy.login(redirect...)
  });

it.doesTheTest....
```

were failing more. We should probably look at why. 

I noticed it because I made a change to the Events page graphql query by removing `createdByUser` and test 23 wasn't failing, whereas test 24 was. I noticed they were structured differently, so I changed test 24 to be structured like test 23 and it passed. Same thing with the other tests (the archive collective ones) - changing the structure fixed them.

Maybe it's a change to Cypress or maybe it's because we've made progressive changes to how login works (with 2FA and now changing LoggedInUser) but I think calling `cy.login()` then other commands within the test works less well now than `cy.login(redirect...)` in the `before`/`beforeEach` setup.

File: test/cypress/integration/30-archiveCollective.test.js
Comment (sbinlondon) [reply test/cypress/integration/30-archiveCollective.test.js:15]: Ahhhh right sorry, sometimes I forget you don't have to separate the befores for each particular test 🤦‍♀️

**Pull request merged**
